### PR TITLE
Add description about telemetry of builtin functions usage

### PR DESCRIPTION
**Pull request by leiysky**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6731
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: telemetry.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ When the telemetry collection feature is enabled in TiDB, the TiDB cluster colle
 - Deployment characteristics, such as the size of hardware (CPU, memory, disk), TiDB components versions, OS name.
 - The status of query requests in the system, such as the number of query requests and the duration.
 - Component usage, for example, whether the Async Commit feature is in use or not.
+- Builtin functions usage information, including the count of functions used in SQL queries.
```
Comment (dveeden) [inline telemetry.md:27]:
```suggestion
- Usage information of builtin functions, including the average number of functions used per SQL query.
```

```suggestion
- Usage information of builtin functions, including the count of functions used in SQL queries.
```

Comment (leiysky) [reply telemetry.md:27]: Actually it is a count per function. 

I'm not sure how to explain it clearly. What do you think about this?

Comment (dveeden) [reply telemetry.md:27]: Would this be ok?
```suggestion
- Usage information of builtin functions, including per-function statistics on how often the functon gets called. The arguments to the functions are not included.
```

Comment (leiysky) [reply telemetry.md:27]: Nice, I'll modify the part in Chinese version according to this as well.

**Review by dveeden — APPROVED** [review]

**Comment by leiysky** [comment]
@TomShawn PTAL

File: telemetry.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ When the telemetry collection feature is enabled in TiDB, the TiDB cluster colle
 - Deployment characteristics, such as the size of hardware (CPU, memory, disk), TiDB components versions, OS name.
 - The status of query requests in the system, such as the number of query requests and the duration.
 - Component usage, for example, whether the Async Commit feature is in use or not.
+- Usage information of builtin functions, including per-function statistics on how often the functon gets called. The arguments to the functions are not included.
```
Comment (TomShawn) [inline telemetry.md:27]:
```suggestion
- The usage information of built-in functions, including per-function statistics on how often the function gets called. The arguments for the functions are not included.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**